lib/nucore.rb

```diff
@@ -108,6 +108,10 @@ def self.included(base)
       end
       module ClassMethods
 
+        def order_by_asc_nulls_first(field)
+          NUCore::Database.oracle? ? order("#{field} asc nulls first") : order(field)
+        end
+
         def order_by_desc_nulls_first(field)
           NUCore::Database.oracle? ? order("#{field} desc nulls first") : order("-#{field}")
```

Comment:
So nulls first in either case? So the ascending order reversed is not the descending order? (Because nulls are always first). Or am I misreading this?